playwright-tests/helpers/utils.ts

```diff
@@ -86,10 +75,18 @@ export const startSessionWithUser = async (
 ): Promise<StartSessionWithUserResponse> => {
     console.log('startingSession:start');
     const name = `${originalName}`;
-    const filePath = `${test.info().project.testDir}/.auth/${name}.json`;
-    const authFileExists = fs.existsSync(filePath);
+
+    console.log('startingSession:directory:checking');
+    const fileDir = `${test.info().project.testDir}/.auth`;
+    const authDirExists = fs.existsSync(fileDir);
+    if (!authDirExists) {
+        console.log('startingSession:directory:creating');
+        fs.mkdirSync(fileDir);
+    }
```

Comment:
Make the tests a little more resilient and handle the missing path. Makes it much easier to clean these up as you can just delete the entire directory.